android/app/src/androidTest/java/org/ZingoLabs/Zingo/IntegrationTest.kt

```diff
@@ -0,0 +1,4 @@
+package org.ZingoLabs.Zingo
```

Comment:
Why/How is ?splitting? this into two files, permitting tests to run?

Reply:
I'm not a Junit guru, but when you see examples you can noticed that.